setup.cfg

```diff
@@ -0,0 +1,192 @@
+[bdist_wheel]
+universal = 1
+
+[metadata]
+name = farm-haystack
+version = file: VERSION.txt
+url = https://github.com/deepset-ai/haystack
+project_urls =
+    Docs: RTD = https://haystack.deepset.ai/overview/intro
+    CI: GitHub = https://github.com/deepset-ai/haystack/actions
+    GitHub: issues = https://github.com/deepset-ai/haystack/issues
+    GitHub: repo = https://github.com/deepset-ai/haystack
+description = Neural Question Answering & Semantic Search at Scale. Use modern transformer based models like BERT to find answers in large document collections
+long_description = file: README.md
+long_description_content_type = text/markdown
+keywords=
+    QA 
+    Question-Answering 
+    Reader 
+    Retriever 
+    semantic-search 
+    search 
+    BERT 
+    roberta 
+    albert 
+    squad 
+    mrc 
+    transfer-learning 
+    language-model 
+    transformer
+author = Malte Pietsch, Timo Moeller, Branden Chan, Tanay Soni
+author_email = malte.pietsch@deepset.ai
+license = Apache License 2.0
+license_file = LICENSE
+platforms = any
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Science/Research
+    License :: Freely Distributable
+    License :: OSI Approved :: Apache Software License
+    Topic :: Scientific/Engineering :: Artificial Intelligence
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+
+[options]
+use_scm_version = True
+python_requires = >=3.7
+packages = find:
+include_package_data = True
+test_suite = tests
+setup_requires =
+    setuptools >= 46.4.0
+install_requires =
+    importlib-metadata; python_version < '3.8'
+    torch>1.9,<1.11  # pytorch
+    tqdm  # progress bars in model download and training scripts
+    requests  # Used for downloading models over HTTP
+    pydantic  # Validation of the core dataclasses (Document, Label, etc...)
+    scipy>=1.3.2  # for stats in run_classifier
+    scikit-learn>=1.0.0  # for stats in run_classifier
+    seqeval  # Metrics or logging related
+    mlflow<=1.13.1  # Metrics or logging related
+    transformers==4.13.0  # huggingface transformers
+    dill  # pickle extension for (de-)serialization
+    #onnxruntime  # Inference with ONNX models. Install onnxruntime-gpu for Inference on GPUs
+    #onnxruntime_tools  # Inference with ONNX models. Install onnxruntime-gpu for Inference on GPUs
+    psutil
+    pandas
+    langdetect   # for PDF conversions
+    pytesseract==0.3.7  # for PDF conversions using OCR
+    pillow  # for PDF conversions using OCR
+    pdf2image==1.14.0  # for PDF conversions using OCR
```

Comment:
Let's make the above three also an optional group. I think pillow made some trouble on colab in the past and OCR is a rather rare conversion method. 

How about `[preprocessing]`? We could also move `tika` and `azure-ai-formrecognizer==3.2.0b2` there.

Reply:
I would keep python-docx and langdetect in the default though as they are more frequently used. We could also later measure with some anonymous telemetry which nodes are really heavily used and adjust the grouping here.